x/benchmark/proto/cosmos/benchmark/v1/benchmark.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package cosmos.benchmark.v1;
```

buf (1.47.2)

Files with package "cosmos.benchmark.v1" must be within a directory "cosmos/benchmark/v1" relative to root but were in directory "x/benchmark/proto/cosmos/benchmark/v1".

(PACKAGE_DIRECTORY_MATCH)